src/unix/aix.c

```diff
@@ -1,4 +1,5 @@
-/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+/* Copyright libuv project contributors. All rights reserved.
```

Comment:
Update copyright text

Reply:
Can you revert that?  This is an old file with the old copyright.

Reply:
I don't know if you accidentally unreverted this but the old copyright should stay in this file.

Reply:
This is still unfixed, it looks like...